sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/Microsoft.Azure.HDInsight.Job.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <Description>Provides job operations for Microsoft Azure HDInsight.</Description>
+        <AssemblyTitle>Microsoft Azure HDInsight Job Library</AssemblyTitle>
+        <AzureApiTag>HDInsightJobManagementClient_2018-11-01-preview;</AzureApiTag>
+        <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
+        <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+        <VersionPrefix>3.0.0</VersionPrefix>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <PackageReference Include="WindowsAzure.Storage" Version="8.1.4" />
+    </ItemGroup>
+    <!-- Please do not move/edit code below this line -->
+    <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
```

Comment:
I assume this import should no longer be needed. Can you please verify and remove if it isn't?